__tests__/index.spec.js

```diff
@@ -168,8 +168,9 @@ pluginTester({
       }
     },
     {
-      title: "Should not do anything with a global import (to be implemented)",
-      code: `import "./styles.scss"`,
+      title:
+        "Should require ios and android files if they exits (side-effects-only import case)",
+      code: `import styles from "./styles.scss"`,
```

Comment:
Shouldn't this be `code: `import "./styles.scss"`,`  if it's testing a side effects import?

Reply:
Good catch! Yeah, missed that one. Fixed with 589e0f3.